src/PhotonBackground.cpp

```diff
@@ -1,11 +1,16 @@
 #include "crpropa/PhotonBackground.h"
 #include "crpropa/Common.h"
+#include "crpropa/Random.h"
+#include "crpropa/Units.h"
 
 #include <vector>
 #include <fstream>
 #include <stdexcept>
 #include <limits>
 #include <cmath>
+#include <sstream>
+#include <iostream>
```

Comment:
The module should not output o the terminal but use the logger

Reply:
iostream was an unused import and has been removed. By logger use do you mean the use of `KISS_LOG_ERROR << "message";` on **each** occasion or are statements of the kind 

https://github.com/CRPropa/CRPropa3/blob/bd39f844217a02c7b5f4d3f6a67d88d22617650a/src/module/PhotoPionProduction.cpp#L76

allright as well?

Reply:
No, throwing an exception is OK.